### ISSUE TITLE
 have problem to get 76% accuarcy on CIFAR10 dataset

### ISSUE THREAD

**Issue by NinaLLI** [titu1994/DenseNet#17]
Dear
I used following  setting for my network but I can not get 76% accuracy on validation set , the optimizer is 'Adam ' and also I did mean-subtraction as preprocessing for  R, G , B channel separately
.would you please help me ?

       model = Sequential()
       model.add(Conv2D(32, (3, 3), border_mode='same',batch_input_shape=(None,32,32,3)))
        
        
        model.add(Activation('tanh'))
        model.add(Conv2D(32, ( 3, 3),border_mode='same'))

        
       model.add(Activation('tanh'))
       model.add(MaxPooling2D(pool_size=(2, 2)))
       model.add(Dropout(0.25))
       model.add(Conv2D(64, (3, 3),border_mode='same')) 
       model.add(Activation('tanh'))
       model.add(Conv2D(64, ( 3, 3),border_mode='same'))
       model.add(Activation('tanh'))
       model.add(MaxPooling2D(pool_size=(2, 2)))
       model.add(Dropout(0.25))
        
       model.add(Flatten())
       model.add(Dense( 1024,init='normal'))
       model.add(Activation('tanh'))
       model.add(Dropout(0.25))
       model.add(Dense( 10,))
       model.add(Activation('softmax'))

**Comment by titu1994** [comment]
This isn't related to DenseNets but no problem I guess. 

Your network uses tanh activation, swap that for relu. Remove init=normal from the Dense layer after flatten.
